### ISSUE TITLE
Bind to Chrome browser events

### ISSUE THREAD

**Issue by dannypernik** [JoacoEsteban/vind#2]
Very intrigued by this extension! I'm curious if it would be possible to bind browser events to keyboard shortcuts with vind. I'm particularly interested in having more keyboard shortcut support for saved tab groups. The use cases are endless, though. Thank you for your consideration!

**Comment by JoacoEsteban** [comment]
Hey Danny, thanks for reaching out, I'm glad you're liking Vind!
On your suggestion, by browser events you mean things like controlling navigation? Do you have a specific use case in mind?

Anyhow, I'm planning on progressively adding more customization on the actions that one can perform, for now I've been trying to keep it simple, as my main use case is to provide shortcuts for apps/websites that have no support for it.

So I'd love to hear your input!

**Comment by dannypernik** [comment]
Makes sense! Yes, I'm thinking of browser events. Specifically, I have found no extension that enables keyboard shortcuts for the following actions related to saved tab groups:

- Toggle save
- Hide group
- Delete group
- Create new group from selected tabs

![image](https://github.com/user-attachments/assets/d5ae67f0-a5dc-4d41-833d-3c370fdb372d)

I did find an extension that allows me to select multiple tabs with the keyboard, so that made me happy.

**Comment by JoacoEsteban** [comment]
Oh that's interesting, I'd never guessed this use case.
I will try to find if there's any Chrome API that allows for managing groups, that should do the trick.

Is there anything else you'd like Vind to do?

**Comment by dannypernik** [comment]
Thank you for looking into Chrome API. That's all for now!
https://developer.chrome.com/docs/extensions/reference/api/tabGroups
